collect_app/src/main/res/xml/preferences.xml

```diff
@@ -75,6 +75,12 @@
             android:key="font_size"
             android:title="@string/font_size" />
         <CheckBoxPreference
+            android:id="@+id/instance_sync"
+            android:defaultValue="true"
+            android:key="instance_sync"
```

Comment:
That's not a bad idea, @nribeka. But as @shobhitagarwal1612 mentions, it's a systematic pattern. Maybe we should change it across all preferences? Please file an issue if you think so. But anyway, better to match the surrounding code for this PR.